### PR TITLE
first draft of descriptions

### PR DESCRIPTION
**Pull request by andy-d-palmer**
what testing needs to be done on the metadata?

### CODE REVIEW

File: min metadata schema.json [hunk 1]
```diff
@@ -15,7 +16,8 @@
           "type": "string"
         },
         "Email": {
-          "type": "string"
+          "type": "string",
+          "description": "required so we can notify you when your dataset is processed"
```
Comment (lomereiter) [inline min metadata schema.json]: Please don't use second person, as the descriptions might be shown to people other than the uploader.

File: min metadata schema.json [hunk 2]
```diff
@@ -24,69 +26,86 @@
     "Sample_Information": {
       "type": "object",
       "required": ["Organism", "Organism_Part", "Condition"],
+      "description": "Information about the imaging sample.  Please select 'other' and enter '-' if not applicable",
```
Comment (lomereiter) [inline min metadata schema.json]: Also please don't reference things specific to a particular web application.

File: min metadata schema.json [hunk 3]
```diff
@@ -24,69 +26,86 @@
...
           "type": "string"
         },
         "Sample_Growth_Conditions": {
+          "description": "How the organism was grown",
           "type": "string"
         }
       }
     },
     "Sample_Preparation": {
+      "description": "Information about the imaging sample. Please select 'other' and enter '-' if not applicable",
```
Comment (lomereiter) [inline min metadata schema.json]: same here

File: min metadata schema.json [hunk 4]
```diff
@@ -113,12 +133,15 @@
     },
     "metaspace_options":{
       "type": "object",
+      "description": "Processing options for the METASPACE engine",
       "required": ["Metabolite_Database", "Dataset_Name"],
       "properties":{
         "Metabolite_Database": {
+          "description": "Choose a metabolite database to use",
```
Comment (lomereiter) [inline min metadata schema.json]: 'metabolite database used for processing'?

File: min metadata schema.json [hunk 5]
```diff
@@ -113,12 +133,15 @@
...
           "type": "string"
         },
         "Dataset_Name":{
+          "description": "A name for your dataset",
```
Comment (lomereiter) [inline min metadata schema.json]: 'Dataset name' would be clear enough.

File: min metadata schema.json [hunk 6]
```diff
@@ -130,15 +153,19 @@
           "type": "string"
         },
         "Sample_Description_Freetext": {
+          "description": "Additional information about your sample (you can simply paste the relevant methods section from a publication)",
```
Comment (lomereiter) [inline min metadata schema.json]: remove 'your', replace 'you can simply paste' with 'e.g.'?

**Review by lomereiter — CHANGES_REQUESTED** [review]
Thanks, looks good overall, but it shouldn't be tailored for the current upload webapp.

**Review by lomereiter — APPROVED** [review]

**Comment by andy-d-palmer** [comment]
I agree with all your suggestions and I have updated accordinly

**Pull request merged**